package.json

```
@@ -6,7 +6,9 @@
     "install:frontend": "cd frontend && npm install",
     "install:backend": "cd backend && npm install",
     "start:frontend": "cd frontend && npm start",
-    "start:server": "cd backend && npm start"
+    "start:server": "cd backend && npm start",
+    "testGUI": "npx cypress open",
+    "testHeadlessMode":"npx cypress run --headless"
```

_headless_ já é padrão do comando `cypress run`, portanto, adicionar `--headless` se torna redundante.

Além disso, o `npx` não é necessário.

```suggestion
    "testHeadlessMode":"cypress run"
```